src/v/cluster/node_isolation_watcher.cc

```diff
@@ -23,8 +23,11 @@ node_isolation_watcher::node_isolation_watcher(
   ss::sharded<node_status_table>& node_status_table)
   : _metadata_cache(metadata_cache)
   , _health_monitor(health_monitor)
-  , _node_status_table(node_status_table) {
+  , _node_status_table(node_status_table) {}
+
+ss::future<> node_isolation_watcher::start() {
```

Comment:
can we return `void`

Reply:
I wanna follow pattern about start and stop methods